lib/member.rb

```diff
@@ -12,7 +18,11 @@ def initialize(member)
   end
 
   field :name do
-    person[:name]
+    names.first.tidy
+  end
+
+  field :alternate_names do
+    names.drop(1).map(&:tidy).join(';') unless names[1].nil?
```

Comment:
My original comment on this (in the first version of this PR) suggested just coercing `.to_a` to avoid the need for this `unless`, but I'm curious as to why that's even needed. If `names` is the result of a `split` it should always return an array, in which case `names.drop(1).map(&:tidy).join(';')` should Just Work™. Was there a scenario in which you actually needed this `unless`?